pkg/porter/generateManifest_test.go

```diff
@@ -74,6 +76,32 @@ func Test_generateInternalManifest(t *testing.T) {
 	}
 }
 
-func mockPullImage(ctx context.Context, image string) (string, error) {
-	return "sha256:8b92b7269f59e3ed824e811a1ff1ee64f0d44c0218efefada57a4bebc2d7ef6f", nil
+type mockRegistry struct {
```

Comment:
After going back to always pulling (above) is it strictly necessary to make a custom registry mock here, that has an internal field for the image summaries? Can we use static data and the mock functions to accomplish the same thing?

Reply:
Good point, I changed it to use a mock function